_LowLevelCode/cpp/accumulate_cut_c/accumulate_cut_c.h

```diff
@@ -49,8 +49,8 @@ mwSize accumulate_cut(double* s, double* e, double* npix,
     {
         ignore_inf = true;
     }
-    ignore_something = ignore_nan | ignore_inf;
-    ignote_all = ignore_nan & ignore_inf;
+    ignore_something = ignore_nan || ignore_inf;
+    ignote_all = ignore_nan && ignore_inf;
```

Comment:
```suggestion
    ignore_all = ignore_nan && ignore_inf;
```
I know it'sin the old code but this is a good chance to fix the spelling here